system/ee/ExpressionEngine/Addons/comment/mod.comment.php

```diff
@@ -31,16 +31,18 @@ class Comment
     public function __construct()
     {
         $fields = array('name', 'email', 'url', 'location', 'comment');
+        
+            foreach ($fields as $val) {
+                if (isset($_POST[$val])) {
+                    $_POST[$val] = ee()->functions->encode_ee_tags($_POST[$val], true);
 
-        foreach ($fields as $val) {
-            if (isset($_POST[$val])) {
-                $_POST[$val] = ee()->functions->encode_ee_tags($_POST[$val], true);
-
-                if ($val == 'comment') {
-                    $_POST[$val] = ee('Security/XSS')->clean($_POST[$val]);
+                    if ($val == 'comment') {
+                    
+                        $_POST[$val] = ee('Security/XSS')->clean($_POST[$val]);
+                    }
                 }
             }
-        }
+        
```

Comment:
It doesn't look like anything was changed here except the indentation was increased and an additional line was added. Can this be reverted as the changes here are incorrect.